rotkehlchen/assets/converters.py

```diff
@@ -436,6 +436,23 @@ def asset_from_common_identifier(common_identifier: str) -> AssetWithOracles:
     ))
 
 
+def asset_from_htx(htx_name: str) -> AssetWithOracles:
+    """May raise:
+    - DeserializationError
+    - UnsupportedAsset
+    - UnknownAsset
+    """
+    if not isinstance(htx_name, str):
+        raise DeserializationError(f'Got non-string type {type(htx_name)} for HTX asset')
+
+    htx_name = htx_name.upper()
+    return symbol_to_asset_or_token(GlobalDBHandler.get_assetid_from_exchange_name(
+        exchange=Location.KRAKEN,  # TODO: Change this to HTX once we add the mappings
```

Comment:
Because I need to add the mappings to the database and that will take me more time from doing the implementation. Can do it in this PR but need some time